db/iaito

```diff
@@ -5,11 +5,18 @@ R2PM_DESC "[gui] C++/Qt GUI for radare2"
 
 R2PM_INSTALL() {
 	cd src || exit 1
-	export PATH="~/Qt/5.6/clang_64/bin:$PATH"
+	export PATH="$HOME/Qt/5.6/clang_64/bin/:$HOME/Qt/5.6/gcc_64/bin/:$PATH"
```

Comment:
maybe worth checking for paths before enforcing them, also maybe it will be good to support cmake if qmake is not found. this is how to build with cmake:

```
cd src ;  cmake .. -DCMAKE_PREFIX_PATH=/opt/Qt/5.6/gcc_64 ; ${MAKE} 
```